src/components/Results/utils.ts

```diff
@@ -200,15 +200,15 @@ const getClosestFacilities = (locations: Location[], zipcode: string, numOfFacil
     .slice(0, numOfFacilities);
 };
 
-export const getStudyDetailProps = (entry: BundleEntry, zipcode: string): StudyDetailProps => {
+export const getStudyDetailProps = (entry: BundleEntry, zipcode: string, serviceName: string): StudyDetailProps => {
   const { resource, search } = entry;
   // Grab the locations so we only have to do this once
   const locations = getLocations(resource);
 
   return {
     conditions: getConditions(resource),
     trialId: getTrialId(resource),
-    source: getSource(),
+    source: serviceName,
```

Comment:
It looks like this removes the use of `getSource()` (added on line 47, which for whatever reason, I can't add a comment to), which should probably be removed.